README.md

```diff
@@ -6,6 +6,20 @@
 
 Scikit-Learn compatible wrappers for Keras Models.
 
+## Why SciKeras?
+
+SciKeras is derived from and API compatible with `tf.keras.wrappers.scikit_learn`. The original TensorFlow (TF) wrappers are not actively mantained, and [may be deprecated](https://github.com/tensorflow/tensorflow/pull/37201#pullrequestreview-391650001) at some point. In addition, they have many incompatibilities with both the Keras ecosystem and the Scikit-Learn ecosystem. SciKeras attempts to resolve these issues by providing mantained, documented wrappers that are fully compatible with the entire Scikit-Learn and Keras ecosystems. Some advantages over the TF wrappers are:
```

Comment:
> Some advantages over the TF wrappers are:

Is there documentation on how to go from TF wrappers to SciKeras wrappers?

Reply:
Not currently. That would also be a good idea to add here. This stuff should also get added to the RTD docs.

Reply:
> That would also be a good idea to add here

When I visted the RTD documentation, I expected to see the transition guide  at the top of the "getting started" page.

> This stuff should also get added to the RTD docs.

Yes, I agree. I tend to create READMEs that point directly to the HTML documentation. If I put anything on the README, it's very developer oriented (here's how to clone, directory structure, etc).

Reply:
I think the "why SciKeras" section should be in both so that people who discover the package via GitHub aren't confused as to why it even exists before they open the docs.

Reply:
I'm going to merge this as is (without any transition guide work). We have #124 to track the transition guide, and in particular, I would like to implement your suggestion in https://github.com/adriangb/scikeras/issues/124#issuecomment-723143535 since that will reduce the required changes. These changes would go beyond the scope of _this_ PR.